pkg/cloud/cloud.go

```diff
@@ -15,12 +15,12 @@ package cloud
 
 import (
 	"fmt"
+	"sigs.k8s.io/aws-encryption-provider/pkg/httputil"
```

Comment:
Can you either put this back in the 3rd party import group or put it on in its own group?

Reply:
Yes , I saw that later when I made the pull request I will correct it soon .

Reply:
updated the code with right place of imports